src/lib/model.ts

```diff
@@ -48,6 +50,7 @@ export interface Calendar extends Entity {
   name: string;
   type: string;
   description: string;
+  teams: Team[];
```

Comment:
lets asume only one team per calendar for now, also add teamId field, team field should be computed in REST API only